geomstats/geometry/minkowski.py

```diff
@@ -16,51 +17,40 @@ class Minkowski(Manifold):
     """
 
     def __init__(self, dim):
-        super(Minkowski, self).__init__(dim=dim)
+        super(Minkowski, self).__init__(dim=dim,
+                                        ambient_manifold=Euclidean(dim))
         self.metric = MinkowskiMetric(dim)
 
-    def belongs(self, point):
-        """Evaluate if a point belongs to the Minkowski space.
+    def belongs(self, point, atol=gs.atol):
+        """Evaluate if a point belongs to the manifold.
 
         Parameters
         ----------
         point : array-like, shape=[..., dim]
             Point to evaluate.
+        atol : float
+            Unused here.
 
         Returns
         -------
         belongs : array-like, shape=[...,]
-            Boolean evaluating if point belongs to the Minkowski space.
+            Boolean evaluating if point belongs to the manifold.
         """
-        point_dim = point.shape[-1]
-        belongs = point_dim == self.dim
-        if gs.ndim(point) == 2:
-            belongs = gs.tile([belongs], (point.shape[0],))
+        return self.ambient_manifold.belongs(point, atol)
```

Comment:
Yes, but will this be slower?

Reply:
I can check. It avoids duplicating code

Reply:
True

Reply:
Running time is similar so I will make the change